src/collections/_documentation/platforms/javascript/sourcemaps/troubleshooting.md

```diff
@@ -19,36 +25,111 @@ Double-check that your deployed, final JavaScript files have `sourceMappingURL`
 
 Alternately, instead of `sourceMappingURL`, you can set a `SourceMap` HTTP header on your minified file. If this header is present, Sentry will use it to discover the location of your source map.
 
-### Verify artifact names match sourceMappingURL
+### Verify artifact names match `sourceMappingURL` value
 
-When [uploading source maps to Sentry](#uploading-source-maps-to-sentry), you must name your source map files with the same name found in `sourceMappingURL`.
+The `sourceMappingURL` comment on the last line of your bundled or minified JavaScript file tells Sentry (or the browser) where to locate the corresponding source map. This can either be a fully qualified URL, a relative path, or the filename itself. When uploading artifacts to Sentry, you must name your source map files with the value the file resolves to.
 
-For example, if you have the following in a minified application file, `app.min.js`:
+i. e. if your file is similar to:
 
 ```javascript
-//-- end app.min.js
-//# sourceMappingURL=https://example.com/dist/js/app.min.js.map
+    // -- end script.min.js
+    //# sourceMappingURL=script.min.js.map
 ```
 
-Sentry will look for a matching artifact named exactly `https://example.com/dist/js/app.min.js.map`.
+and is hosted at http://example.com/js/script.min.js, then Sentry will look for that source map file at http://example.com/js/script.min.js.map. Your uploaded artifact must therefore be named `http://example.com/js/script.min.js.map` (or `~/js/script.min.js.map`). 
```

Comment:
@benvinegar this wording was my suggested revision - can you verify that it accurately reflects reality?

Reply:
Looks correct.